src/slic3r/GUI/ConfigManipulation.cpp

```diff
@@ -320,6 +321,30 @@ void ConfigManipulation::update_print_fff_config(DynamicPrintConfig* config, con
         is_msg_dlg_already_exist = false;
     }
 
+    bool have_arachne = config->opt_enum<PerimeterGeneratorType>("wall_generator") == PerimeterGeneratorType::Arachne;
+	if (!is_plate_config &&
+        config->opt_bool("staggered_perimeters") &&
+        (abs(config->opt_float("initial_layer_print_height") - config->opt_float("layer_height")) > EPSILON  ||
+        !(config->option<ConfigOptionFloatOrPercent>("top_surface_line_width") == config->option<ConfigOptionFloatOrPercent>("outer_wall_line_width")) || 
```

Comment:
Thanks for your work on this! FYI the comparison here is comparing the pointers, and not using the overridden equality. This makes the popup warning appear every time basically. Maybe using the `opt_float` would work here too.
```suggestion
        !(config->option<ConfigOptionFloatOrPercent>("top_surface_line_width")->operator==(*config->option<ConfigOptionFloatOrPercent>("outer_wall_line_width"))) ||
```